__tests__/app.test.js

```diff
@@ -53,27 +53,133 @@ describe("GET requests", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          const { review } = body;
+          expect(review).toEqual({
+            review_id: 2,
+            title: "Jenga",
+            designer: "Leslie Scott",
+            owner: "philippaclaire9",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            review_body: "Fiddly fun for all the family",
+            category: "dexterity",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 5,
+          });
+        });
+    });
+    describe("Error handlers", () => {
+      test("Status: 404, given a valid but nonexistent id returns a 'nonexistent id' msg", () => {
+        return request(app)
+          .get("/api/reviews/33")
+          .expect(404)
+          .then(({ body }) => {
+            expect(body.msg).toEqual("nonexistent id");
+          });
+      });
+      test("Status: 400, given a invalid id returns a 'invalid id' msg", () => {
+        return request(app)
+          .get("/api/reviews/banana")
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toEqual("invalid request");
+          });
+      });
+    });
+  });
+
+  describe("GET /api/reviews/:review_id/comments", () => {
+    test("Status: 200, given a review id returns an array of all comment objects with that review id", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toHaveLength(3);
+          expect(comments).toEqual([
+            {
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              votes: expect.any(Number),
+              author: expect.any(String),
+              review_id: expect.any(Number),
+              created_at: expect.any(String),
+            },
+            {
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              votes: expect.any(Number),
+              author: expect.any(String),
+              review_id: expect.any(Number),
+              created_at: expect.any(String),
+            },
+            {
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              votes: expect.any(Number),
+              author: expect.any(String),
+              review_id: expect.any(Number),
+              created_at: expect.any(String),
+            },
+          ]);
+        });
+    });
+    test("Status: 200, returns comment objects sorted by date in descending order", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeSortedBy("created_at", { descending: true });
+        });
+    });
```

Comment:
We could consider an extra test here, what if the request is made for a valid and existent id but there happens to be no comments? We shouldn't get an error in that case. Lets write a test for it and then consider how we can differentiate between no results legitimately and no results for an error reason.